blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -139,30 +139,30 @@ builder.addGlobalStore(
 
 Remember that all store operations (like get) will lead to deserialization that costs you some CPU.
 
 
 ## Many Source Topics Within a Kafka Stream
 
-At first, we designed Kestra to have only one **huge** stream for all the processing from the executor. At first, it seemed cool, but this led to some drawbacks.
+At first, we designed Kestra to have only one **huge** stream for all the processing of the executor. At first, it seemed cool, but this led to some drawbacks.
 
 Here is the last version of our main and only Kafka Stream with many topics 🙉:
 ![Kestra Topology](./2022-06-28-tips-kafka-streams-distributed/topology.jpg)
-Yes, this is a huge Kafka Stream. It was working almost despite the complexity of this one. But the major drawbacks were :
-- **Monitoring**: All the metrics are under the same consumer groups
+Yes, this is a huge Kafka Stream. It was working well despite its complexity. But the major drawbacks were :
+- **Monitoring**: All the metrics are under the same consumer group.
 - **Debugging**: Each topic is consumed independently during a crash. When a message fails, the whole process crashes.
 - **Lag**: This is the most important one. Since Kafka Streams optimize the consumption of messages by themselves, a topic with large outputs could lead to lag on unrelated topics. In that case, it is impossible to understand the lag on our consumers.
 
-Now, we have decided to split into multiple [streams](https://github.com/kestra-io/kestra/tree/92a856fa5b3fa3a29d251ec9873f631caa2678a0/runner-kafka/src/main/java/io/kestra/runner/kafka/executors) to be able to monitor and properly understand the lag on our Kafka Streams.
+Now, we have decided to split it into multiple [streams](https://github.com/kestra-io/kestra/tree/92a856fa5b3fa3a29d251ec9873f631caa2678a0/runner-kafka/src/main/java/io/kestra/runner/kafka/executors) to be able to monitor and properly understand the lag on our Kafka Streams.
 
 To split our giant stream, we dealt with only one topic at a time. We consumed only one topic at a time (to avoid large network transit), so we grouped all streams by source topics.
 
 
 ## Do More With Kafka Streams
 
-We have covered some tips that took us a lot of time to find how to deal with the issues. Even though there were some challenges, we could adapt our code so that Kafka worked well for our use case.
+We have covered some tips that took us a lot of time to find in order to deal with our issues. Even though there were some challenges, we could adapt our code so that Kafka worked well for our use case.
```

Comment:
```suggestion
We have covered some tips that took us a lot of time to find to deal with our issues. Even though there were some challenges, we could adapt our code so that Kafka worked well for our use case.
```

Conciseness